server/src/test/java/io/crate/execution/engine/aggregation/impl/MaximumAggregationTest.java

```diff
@@ -58,6 +60,13 @@ public void test_function_implements_doc_values_aggregator_for_numeric_types() {
         }
     }
 
+    @Test
+    public void testNumeric() throws Exception {
+        Object result = executeAggregation(new NumericType(6, 4),
+            new Object[][]{{new BigDecimal("97.6543")}, {new BigDecimal("97.6542")}});
+        assertThat(result).isEqualTo(new BigDecimal("97.6543"));
+    }
```

Comment:
Can we try input with a different precision?

I mean testing scenario `min(97.12345678, 97.987654321) : Numeric (4,2)`

Reply:
exactly, I don't think we need to add this, since we test the return type explicitly in the other class.

Reply:
>I wonder if we should register as in prod, ie use

That wouldn't work, okay, it's a bit artificial binding here.